sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/unresolved.scala

```diff
@@ -88,12 +88,12 @@ case class UnresolvedAttribute(nameParts: Seq[String]) extends Attribute with Un
   override def exprId: ExprId = throw new UnresolvedException(this, "exprId")
   override def dataType: DataType = throw new UnresolvedException(this, "dataType")
   override def nullable: Boolean = throw new UnresolvedException(this, "nullable")
-  override def qualifier: Option[String] = throw new UnresolvedException(this, "qualifier")
+  override def qualifier: Option[Seq[String]] = throw new UnresolvedException(this, "qualifier")
```

Comment:
I think we can just use `Seq[String]`

Reply:
I followed the style that exists today. If we change here, then there are other places where it needs to change as well.  For e.g, in AttributeReference we initialize it with None today and now it will be a Seq.empty.  I am not sure if we want to create these objects or just leave it using  None by keeping the Option[]  

If you have a strong preference, I can update it.   I do have the changes locally but have not pushed that version.   Thanks.